docs/README.md

```diff
@@ -3,3 +3,60 @@
 Caltrans Performance Measurement System
 
 PeMS is open-source software that is designed, developed, and maintained by [Compiler LLC](https://compiler.la) on behalf of Caltrans Traffic Operations.
+
+## Current Work
+We do sprint planning and track day-to-day work in our [Project Board](/https://github.com/orgs/compilerla/projects/3/views/1).
+
+See our current [Milestones](/https://github.com/compilerla/pems/milestones) for a higher level view of project progress.
+
+## Product Roadmap
+Our product roadmap captures what we're currently building, what we've built, and what we plan to build in future quarters. It is updated on a regular basis, aligned with project progress.
+
+```mermaid
+timeline
+    title PeMS Product Roadmap
+%% PeMS Epics (2024)
+          section 2024
+
+          Q3<br>Complete
+          : Project launch
+          : Established resources and overall scope
+          : Began discovery work
+
+          Q4<br>Now
+          : Confirm site architecture
+          : Create local environment
+          : Set up GitHub repository
+          : Scaffold app structure
+          : Create a prioritized backlog of features
+
+%% PeMS Epics (2025)
```

Comment:
I think you'll want to update this section, seems like a copy of the Benefits 2025 roadmap?

Reply:
We had both of these as Q4 items in our google docs roadmap, but I can take em out no problem - lemme know: https://docs.google.com/document/d/18umi2b7TuszcSlwXB0X6gAwSBzxYUUFrcub6cW7lrb4/edit?usp=sharing

Reply:
Or maybe my text updates didn't work? I did copy Andy's markdown, but I thought the text was updated to PeMS (at least it was for me?)

Reply:
jk i see it now lol - fixing!